core/src/commonMain/kotlin/org/jetbrains/litmuskt/LitmusTest.kt

```diff
@@ -18,20 +19,28 @@ class LitmusTestScope<S : Any>(
     private lateinit var outcomeFinalizer: S.() -> LitmusOutcome
     private lateinit var outcomeSpec: LitmusOutcomeSpecScope<S>
 
+    // WARNING: we cannot use `(S as LitmusAutoOutcome).reset()` here. DO NOT FORGET to call it in the runner!
```

Comment:
Actually, why can't we do this? 
Taking into account the fact that outcome might not necessary be `LitmusAutoOutcome` inheritor:
```kotlin
(this as? LitmusAutoOutcome)?.reset()
```

It would save as from the necessity to call `outcomeReset()` in every `reset` block of DSL in tests.

Reply:
We absolutely can and it makes it so much better! (Turns out I still forgot to call `outcomeReset` in one test :facepalm: ). Apparently I tunnel-visioned on getting a reference like a `S::outcomeReset`, which we indeed cannot obtain here. Thanks for the suggestion! :star: